src/StyledFormComponents.js

```diff
@@ -17,12 +17,14 @@ const StyledCardHeading = styled.h3`
 `;
 
 const StyledCard = styled.div`
+  width: ${({ favouritesCard }) => (favouritesCard ? `45%` : `900px`)};
+  height: ${({ favouritesCard }) => (favouritesCard ? `80vh` : `450px`)};
+  margin: ${({ favouritesCard }) => (favouritesCard ? `10px` : `0 auto`)};
+  display: ${({ favouritesCard }) => (favouritesCard ? `inline-block` : ``)};
```

Comment:
This last one is probably better as  ${({ favouritesCard }) => favouritesCard && 'display: inline-block;'};

Reply:
should remove the margin properity from line 33 too